sdk/include/crt/crtdefs.h

```diff
@@ -438,10 +438,13 @@ typedef struct threadmbcinfostruct *pthreadmbcinfo;
 
 struct __lc_time_data;
 
+#ifndef DEFINED_localeinfo_struct
 typedef struct localeinfo_struct {
     pthreadlocinfo locinfo;
     pthreadmbcinfo mbcinfo;
 }_locale_tstruct,*_locale_t;
+#define DEFINED_localeinfo_struct 1
```

Comment:
A value seems superfluous/misleading:
```suggestion
#define DEFINED_localeinfo_struct
```

Reply:
Submitted as #6769.